lisp/forge-core.el

```diff
@@ -75,7 +75,10 @@
     ("git.suckless.org" nil
      "git.suckless.org" forge-stagit-repository)
     ("git.sr.ht" nil
-     "git.sr.ht" forge-srht-repository))
+     "git.sr.ht" forge-srht-repository)
+    ;; gitee (API unsupported)
```

Comment:
Put this below the bitbucket entry into the "Forges (API unsupported)" group. (Don't add an additional comment, the comment I just mentioned covers that.)